Client/Frontend/Home/JumpBackIn/Data/JumpBackInDataAdaptor.swift

```diff
@@ -224,6 +224,9 @@ class JumpBackInDataAdaptorImplementation: JumpBackInDataAdaptor, FeatureFlaggab
 
         // Get cached tabs
         userInteractiveQueue.async { [weak self] in
+            /// Force an accounts Sync so that we download new records to local storage (also uploads records from local storage to the sync server)
+            _ = self?.profile.syncManager.syncNamedCollections(why: .user, names: ["tabs"])
```

Comment:
Let's remove this call because the tab event changes trigger enough syncs. Also, this one gets called once when we start the app. 

Definitely remove this.

Reply:
My only worry is there will be way too many calls with tab event changes. Instead we should look into simply when tab is finished loading

Reply:
And I feel even that might be a lot depending on how many times a user is navigating tab